packages/admin/src/stack/Switch.tsx

```diff
@@ -49,6 +53,9 @@ export function useStackSwitch(): [React.ComponentType<IProps>, IStackSwitchApi]
         activatePage: (pageName: string, payload: string, subUrl?: string) => {
             apiRef.current?.activatePage(pageName, payload, subUrl);
         },
+        getTargetUrl: (pageName: string, payload: string, subUrl?: string) => {
+            return apiRef.current?.getTargetUrl(pageName, payload, subUrl) || "";
```

Comment:
Is this a good way of handling the "undefined"-case or should I rather throw an exception?

Reply:
good question. Or return "" and log an console error.